qpmodel/stmt.cs

```diff
@@ -115,30 +115,22 @@ public virtual List<Row> Exec()
 
         public static List<Row> ExecSQL(string sql, out SQLStatement stmt, out string physicplan, out string error, QueryOption option = null)
         {
-            try
-            {
-                stmt = RawParser.ParseSingleSqlStatement(sql);
-                if (option != null)
-                    stmt.queryOpt_ = option;
-                var result = stmt.Exec();
-                physicplan = "";
-                if (stmt.physicPlan_ != null)
-                    physicplan = stmt.physicPlan_.Explain(option?.explain_);
-                error = "";
-                return result;
-            }
-            catch (Exception e)
```

Comment:
Catch everything and return null silently makes debugging super hard. I'm making it more explicitly but if @zhouqingqing have other concerns I can switch it back.

Reply:
agree try/catch is bad here. We need try/catch to catch "expected" error, say you write "select select 1;", it is an expected syntax error. We do have some tests to verify syntax/algebraic/exec errors (SemanticAnalyzeExeception and SemanticExecutionException).

What's causing problems here are unexpected errors. So a reasonable fix is to check errors in catch block and throw unexpected errors.